README.md

```diff
@@ -1,2 +1,128 @@
-# PotatoORM
-A simple agnostic ORM that can perform the basic crud database operations.
```

Comment:
**Nitpick:** "CRUD" should be in capitals. Will also be good to give some context about this -- a _Checkpoint 2_ for Andela Fellowship.